### PR TITLE
Feature/generate open api spec

### PR DESCRIPTION
**Pull request by michelu89**
A new icon in the toolbar makes it possible to generate an open api from the valid aspect model.

### CODE REVIEW

File: core/apps/ame-e2e/src/fixtures/valid-documentation.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!doctype html>
```
Comment (georgschmidtdumont) [inline core/apps/ame-e2e/src/fixtures/valid-documentation.html:1]: Please add the copyright header.

File: core/apps/ame-e2e/src/fixtures/valid-open-api.json [hunk 2]
```diff
@@ -0,0 +1,154 @@
+{
```
Comment (georgschmidtdumont) [inline core/apps/ame-e2e/src/fixtures/valid-open-api.json:1]: Please add the copyright header.

File: core/apps/ame-e2e/src/fixtures/valid-open-api.yaml [hunk 3]
```diff
@@ -0,0 +1,99 @@
+---
```
Comment (georgschmidtdumont) [inline core/apps/ame-e2e/src/fixtures/valid-open-api.yaml:1]: Please add the copyright header.

**Review by georgschmidtdumont — CHANGES_REQUESTED** [review]

File: core/apps/ame-e2e/src/fixtures/valid-open-api.json (hunk 2 above)
Comment (michelu89) [reply core/apps/ame-e2e/src/fixtures/valid-open-api.json:1]: Cannot include copyright to json

**Review by georgschmidtdumont — APPROVED** [review]

**Pull request merged**